packages/react/src/hooks/usePopoverPositioning.test.jsx

```diff
@@ -88,7 +88,7 @@ describe('usePopoverPositioning', () => {
     userEvent.click(button);
     const menu = screen.getByTestId(testId);
 
-    expect(menu).toHaveClass(`${iotPrefix}--flyout-menu--body__${FlyoutMenuDirection.LeftEnd}`);
```

Comment:
This test was incorrect and is changed, because of the flyoutAlignment changes that fixes an edge case above: https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3133/files#diff-c56a39e4778901918956ade07124779426ef2c20782d0ac1bdc7d7981805c08cR205